libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -104,15 +104,21 @@ def _merge_messages(
     for curr in messages:
         curr = curr.copy(deep=True)
         if isinstance(curr, ToolMessage):
-            curr = HumanMessage(  # type: ignore[misc]
-                [
-                    {
-                        "type": "tool_result",
-                        "content": curr.content,
-                        "tool_use_id": curr.tool_call_id,
-                    }
-                ]
-            )
+            if isinstance(curr.content, list) and any(
```

Comment:
uber nit but shouldn't these all be tool_result type (instead of `any`)?

Reply:
good catch thanks.